lib/tasks/analyse/links.rb

```diff
@@ -167,9 +174,13 @@ def add_rows(data:, start_row_number:, worksheet:)
     end
   end
 
-  def results_as_arrays(results)
+  def results_as_arrays(results, worksheet)
     results.map do |result|
-      result_columns.map { |column| result[column] }
+      if worksheet == "Overview"
```

Comment:
Again, I find this "magic string" approach a bit surprising, and it's generally fragile. Again, it might be nicer to pass in the array of desired columns, and then you can do away with the `if` statement.